knimin/lib/tests/test_data_access.py

```diff
@@ -329,6 +328,23 @@ def test_get_ag_barcode_details(self):
             self.assertEqual({k: obs[key][k] for k in exp[key]}, exp[key])
             self.assertIn(obs[key]['participant_name'], participant_names)
 
+    def test_list_ag_surveys(self):
+        truth = [(-1, 'Personal Information', True),
```

Comment:
The test would not be correct, since it will not be testing that all surveys are returned.